frontera.md

```diff
@@ -12,18 +12,19 @@ Then execute:
 
 ```
 module load tacc-apptainer
-apptainer pull -F docker://seissol/training:hps-2024-frontera
+apptainer pull -F docker://seissol/training:latest
 apptainer build -f my-training.sif singularity.def
```

Comment:
Pull and build seem to be duplicate here.

Reply:
ok so we should remove the line `apptainer pull -F docker://seissol/training:latest` ?